src/bin/commands/new-project/utils/imports.ts

```diff
@@ -8,33 +8,66 @@ interface Import {
 }
 
 export class Imports {
-  imports: Record<string, Import>;
+  private readonly imports: Record<string, Import>;
 
-  constructor(imports?: Record<string, Import>) {
-    this.imports = imports ?? {};
+  private constructor(imports: Record<string, Import>) {
```

Comment:
Although `Imports` is exported, instantiation is only performed by:
  - `newStackImports`
  - `newAppImports`
  - `newTestImports`

In this PR we make `Imports`'s constructor private, and convert the aforementioned functions to static methods on the class to improve namespacing.